[BUG] cugraph build broken for CentOS-7

**Describe the bug**
After adding cugraph to the nightly container builds, the CentOS-7 container fails to build. However, Ubuntu versions build successfully.  The error is:
```
[ 75%] Linking CXX executable ../../gtests/JACCARD_TEST
/usr/bin/ld: cannot find -lgmock_main
/usr/bin/ld: cannot find -lgmock
collect2: error: ld returned 1 exit status
```

**To Reproduce**
Using a CentOS-7 system -preferably one of the nightly containers- attempt to build cugraph. 

**Expected behavior**
A successful build, as seen on a Ubuntu system.

**Desktop (please complete the following information):**
 - CentOS-7 64-bit

**Additional context**
On CentOS, the gtest dependency is built and installed to `/rapids/cugraph/cpp/build/CMakeFiles/thirdparty/googletest-install/lib64`, but the cmake-generated `CMakeFiles/RENUMBERING_TEST.dir/link.txt` file specifies `-L/rapids/cugraph/cpp/build/CMakeFiles/thirdparty/googletest-install/lib` (not `lib64`). I haven't checked Ubuntu, but I'm guessing it ends up in `lib` and everything is found.  Still looking, but I'm assuming something needs to be tweaked in `cpp/src/tests/CMakeLists.txt` for the `target_link_libraries` call in the `configure_tests` function, or we need to somehow force it to install to `lib`...not sure yet though.  I tested this theory by manually editing `link.txt` and changing `lib` to `lib64` and the test executable links successfully.

One workaround was to edit `cpp/cmake/Modules/ConfigureGoogleTest.cmake` and add the following line at the end of the file:
```
link_directories(${GTEST_ROOT}/lib64/)
```

It should also be noted that `link_directories` _is rarely necessary. Library locations returned by find_package() and find_library() are absolute paths. Pass these absolute library file paths directly to the target_link_libraries() command. CMake will ensure the linker finds them._ https://cmake.org/cmake/help/v3.0/command/link_directories.html

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -40,6 +40,7 @@
 - PR #120 Bug fix for segfault calling spectral clustering with only edge list
 - PR #123 Fixed weighted Jaccard to assume the input weights are given as a cudf.Series
 - PR #152 Fix conda package version string
+- PR #160 Added additional link directory to support building on CentOS-7
 
 
 # cuGraph 0.5.0 (28 Jan 2019)

--- a/cpp/cmake/Modules/ConfigureGoogleTest.cmake
+++ b/cpp/cmake/Modules/ConfigureGoogleTest.cmake
@@ -64,3 +64,5 @@ set(GTEST_ROOT ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/thirdparty/googletest
 message(STATUS "GTEST_ROOT: " ${GTEST_ROOT})
 
 link_directories(${GTEST_ROOT}/lib/)
+# FIXME: lib64 also needs to be added for CentOS-7. This might be a code smell - investigate.
+link_directories(${GTEST_ROOT}/lib64/)

